src/crystal/system/system.cr

```diff
@@ -0,0 +1,24 @@
+# :nodoc
```

Comment:
We don't want to `:nodoc:` `Crystal` as we loose documenting macros and `Crystal::VERSION`. The `:nodoc:` should be placed on `Crystal::System` only.